pkg/services/system_intakes.go

```diff
@@ -109,3 +112,37 @@ func NewFetchSystemIntakeByID(
 	}
 
 }
+
+// NewSubmitSystemIntake is a service to submit system intake to CEDAR
+func NewSubmitSystemIntake(
+	submit func(intake *models.SystemIntake, logger *zap.Logger) (string, error),
+	save func(intake *models.SystemIntake) error,
+	logger *zap.Logger,
+) func(intake *models.SystemIntake, logger *zap.Logger) error {
+	return func(intake *models.SystemIntake, logger *zap.Logger) error {
```

Comment:
This need to be authorized the same way that a `save` is. Otherwise, someone could run this code without having previous ownership of this intake and it will both submit and save. My guess is this code might end up looking a lot like `NewSaveSystemIntake`. It's possible you'll just want to combine the two and conditionally `submit`. Not 100% on the latter part, but it seems the cleanest, as you can also get rid of the new code in `handlers` and just branch on `SUBMITTED` here.